### PR TITLE
Redo - Higher histogramm range uint16

### LINKED ISSUE #4354: Min/Max default for uint16
Currently, the min/max in the historgram defaults to 0-256 for uint16 datasets. The range is corrected after the histogram sampling is complete. However, it would be nice to have the default be 0-65535.

### PR DESCRIPTION
**Pull request by MichaelBuessemeyer**
This PR changes the default intensity range for uint16 data layers to the interval [0, 65535]. Previously it was [0, 255], causing uint16 data layers to be almost completely white by default. Now the data of uint16 layers should be visible by default.

### URL of deployed dev instance (used for testing):
- https://higherhistogrammrangeuint.webknossos.xyz

### Steps to test:
- Open a dataset with a uint16 data layer. The data should initially visible without having to adjust the histogram.

### Issues:
- fixes #4354

------
- ~~[ ] Maybe write saga test(s).~~
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- ~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~
- ~~[ ] Updated [documentation](../blob/master/docs) if applicable~~
- ~~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~~
- ~~[ ] Needs datastore update after deployment~~
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/oxalis/model/accessors/dataset_accessor.js [hunk 1]
```diff
@@ -103,6 +103,40 @@ export function getElementClass(dataset: APIDataset, layerName: string): Element
...
+      return [0, Math.pow(2, 64) - 1];
+    // We do not fully support signed int data;
+    case "int16":
+      return [0, Math.pow(2, 15) - 1];
+    case "int32":
+      return [0, Math.pow(2, 31) - 1];
+    case "int64":
+      return [0, Math.pow(2, 63) - 1];
+    case "float":
+      return [0, maxFloatValue];
```
Comment (MichaelBuessemeyer) [inline frontend/javascripts/oxalis/model/accessors/dataset_accessor.js]: @philippotto I reopened a PR for the better default intensityRange values.

Strangely, the screenshot tests do not fail for me (locally with `yarn test-screenshot`).
Nevertheless, I am pretty sure why the screenshot tests failed: The previous default intensityRange for float was `[0;255]`. With this PR the max value will become much larger, causing the data to be black.

The core issue here, in my opinion, is that we do did not fully agree on a "good" default value for floating-point data types.  
The high max value of `3.40282347e38` only makes sense when this is the actual max value for most of the float data layers. 
I think almost all float data layers have the range of [0, 255], thus we should treat float types the way we did before.
I hope you get my point, but let's talk about this in the office.

Comment (MichaelBuessemeyer) [inline frontend/javascripts/oxalis/model/accessors/dataset_accessor.js]: @philippotto I reopened a PR for the better default intensityRange values.

Strangely, the screenshot tests do not fail for me (locally with `yarn test-screenshot`).
Nevertheless, I am pretty sure why the screenshot tests failed: The previous default intensityRange for float was `[0;255]`. With this PR the max value will become much larger, causing the data to be black.

The core issue here, in my opinion, is that we do did not fully agree on a "good" default value for floating-point data types.  
The high max value of `3.40282347e38` only makes sense when this is the actual max value for most of the float data layers. 
I think almost all float data layers have the range of [0, 255], thus we should treat float types the way we did before.
I hope you get my point, but let's talk about this in the office.

**Comment by MichaelBuessemeyer** [comment]
@philippotto Could you please review this PR?

Do you think I should add a test for the histogram fetching saga?

**Comment by MichaelBuessemeyer** [comment]
@philippotto Thanks for your feedback. Some tests are still failing. Do you maybe know why? Let's talk about this tomorrow :)

**Comment by MichaelBuessemeyer** [comment]
> However, I just tested this branch and it does not seem to work, yet? When I open https://higherhistogrammrangeuint.webknossos.xyz/datasets/sample_organization/float_test_dataset/view#904,594,8,0,1.000 the data is black and only renders properly if I open the settings?

I just tried it out too but it seems to work for me :thinking:

**Comment by MichaelBuessemeyer** [comment]
Strangely tests still fail on my maschine. @philippotto Could you please check the new changes again?

**Comment by philippotto** [comment]
> > However, I just tested this branch and it does not seem to work, yet? When I open https://higherhistogrammrangeuint.webknossos.xyz/datasets/sample_organization/float_test_dataset/view#904,594,8,0,1.000 the data is black and only renders properly if I open the settings?
> 
> I just tried it out too but it seems to work for me

Hm, maybe it was a caching issue. It works for me now, too :)

**Comment by MichaelBuessemeyer** [comment]
Sure, I am on it :+1:

**Pull request merged**